alembic/versions/73166af50deb_fix_trunk_with_host_migration.py

```diff
@@ -0,0 +1,133 @@
+"""fix trunk with host migration
+
+Revision ID: 73166af50deb
+Revises: f98e74435092
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '73166af50deb'
+down_revision = 'f98e74435092'
+
+user_sip_tbl = sa.sql.table(
+    'usersip',
+    sa.sql.column('name'),
+    sa.sql.column('category'),
+    sa.sql.column('host'),
+    sa.sql.column('username'),
+    sa.sql.column('port'),
+)
+endpoint_sip_tbl = sa.sql.table(
+    'endpoint_sip',
+    sa.sql.column('uuid'),
+    sa.sql.column('name'),
+)
+endpoint_sip_section_tbl = sa.sql.table(
+    'endpoint_sip_section',
+    sa.sql.column('uuid'),
+    sa.sql.column('type'),
+    sa.sql.column('endpoint_sip_uuid'),
+)
+endpoint_sip_section_option_tbl = sa.sql.table(
+    'endpoint_sip_section_option',
+    sa.sql.column('uuid'),
+    sa.sql.column('key'),
+    sa.sql.column('value'),
+    sa.sql.column('endpoint_sip_section_uuid'),
+)
+
+
+def get_trunk_with_host():
+    query = sa.sql.select([
+        user_sip_tbl.c.name,
+        user_sip_tbl.c.host,
+        user_sip_tbl.c.username,
+        user_sip_tbl.c.port,
+    ]).where(
+        sa.and_(
+            user_sip_tbl.c.category == 'trunk',
+            user_sip_tbl.c.host != 'dynamic',
+        )
+    )
+    rows = op.get_bind().execute(query)
+    return [{
+        'name': row.name,
+        'host': row.host,
+        'username': row.username,
+        'port': row.port,
+    } for row in rows]
+
+
+def convert_host_to_contact(trunk):
+    result = 'sip:'
+
+    username = trunk.get('username')
+    if username:
+        result += username + '@'
+
+    port = trunk.get('port', '5060')
```

Comment:
```suggestion
    port = trunk.get('port', '5060', '')
```
Otherwise you'll have a `None` in string :grin: